eval.py

```diff
@@ -98,7 +100,7 @@ def decisions(agent, data, window_size, debug, stock, api):
 
                     # Countdown timer until market opens
                     while time_to_open > -1:
-                        print(str(time_to_open) + " minutes til market open.", end='\r')
+                        print(str(time_to_open) + " minutes til market open.", end='\n')
```

Comment:
`\r`
makes it so it won't constantly make new lines every minute, if you leave it, there won't be a carriage return and it will rewrite the last line. This gives the effect of a countdown timer of sorts, I definitely prefer that.

Reply:
Oh, yeah, I didn't mean to put that there that's just my personal preference.